### PR TITLE
Renamed the package to ezplatform-richtext

### PR DESCRIPTION
**Pull request by bdunogier**
> [EZP-29058](https://jira.ez.no/browse/EZP-29058)

Title and story say it all.

### CODE REVIEW

File: docs/Structure.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# RichText structure
```
Comment (alongosz) [inline docs/Structure.md]: I'd rather handle such small portion of information in `README.md` (see also other comments for this file).
Or is there a wider plan for this doc?

File: log.txt [hunk 2]
```diff
@@ -0,0 +1,40 @@
+236ceeb12883902ce119a42921d5cabd10af7b1a [Structure] Applied php-cs, and enabled in travis
```
Comment (alongosz) [inline log.txt]: Maybe let's rename this file to something more meaningful, like `ezpublish-kernel.git.log`.
It's just that if I saw log.txt in the main directory of any repo I'd think that someone commited it by accident :upside_down_face:

File: config.php [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+/**
+ * eZ Publish 5.x config.php file.
+ *
+ * Returns global application settings
+ * Usually contain settings needed to setup services needed for ezp startup.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @doc Copy this file to config.php to get started!
```
[the remark this answers is not in the document]
Comment (bdunogier) [reply config.php]: Actually, it should not have been added, as it gets copied by the integration tests :)

File: log.txt (hunk 2 above)
Comment (bdunogier) [reply log.txt]: Not meant to be here. I may have used a brutal git add 🗡

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+| Question           | Answer
+| ------------------ | ------------------
+| **JIRA issue**     | [EZP-XXXXX](https://jira.ez.no/browse/EZP-XXXXX)
+| **Bug/Improvement**| yes/no
+| **New feature**    | yes/no
+| **Target version** | `6.x`/`7.x` for bug fixes or improvements _(on existing features)_, `master` for features
```
[the remark this answers is not in the document]
Comment (bdunogier) [reply .github/pull_request_template.md]: Done.

File: docs/Structure.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+
+## ezpublish-kernel extraction
+This package was extracted from ezsystems/ezpublish-kernel, and remapped to a new structure in its own package.
```
[the remark this answers is not in the document]
Comment (bdunogier) [reply docs/Structure.md]: This file wasn't supposed to be added, I'll remove it.

**Pull request merged**